specutils/analysis/location.py

```diff
@@ -19,7 +21,9 @@ def centroid(spectrum, region):
     Parameters
     ----------
     spectrum : `~specutils.spectra.spectrum1d.Spectrum1D`
-        The spectrum object overwhich the centroid will be calculated.
+        The spectrum object overwhich the centroid will be calculated.  If the uncertainty
```

Comment:
```suggestion
        The spectrum object over which the centroid will be calculated.  If the uncertainty
```
(minor pre-existing typo)